machine-learning/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Ignore the data folder that will be symlinked on the RStudio Server or, in the
+# case of workshops run on personal laptops, distributed to participants
+data/
```

Comment:
I left the symlinks as symlinks that will be broken if downloaded to a different machine.... your way is probably better.